tests/RateLimitErrorTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Stripe;
+
+class RateLimitErrorTest extends TestCase
+{
+    private function rateLimitErrorResponse()
+    {
+        return array(
+            'error' => array(),
+        );
+    }
+
+    /**
+     * @expectedException Error\RateLimit
+     */
+    public function testRateLimit()
```

Comment:
This test isn't correct. I'd split this up into two tests.

```php
**
 * @expectedException Error\RateLimit
 */
public function testRateLimit()
{
    $this->mockRequest('GET', '/v1/accounts/acct_DEF', array(), $this->rateLimitErrorResponse(), 429);
    Account::retrieve('acct_DEF');
}
```
and another than just checks that `Error\RateLimit` returns 429 for `getHttpStatus`.